test/JDBC/expected/BABEL-3952-vu-prepare.out

```diff
@@ -0,0 +1,515 @@
+-- Test with null datepart
+-- Should Throw Error - 'syntax error at or near "null"' (error from parser side)
+-- VIEW DATE_BUCKET_vu_prepare_v1 DATE_BUCKET_vu_prepare_v2 should not get created.
+CREATE VIEW DATE_BUCKET_vu_prepare_v1 AS (
+    select 
+        date_bucket(null, 2, cast('2020-01-01' as date)) as db
+    );
+GO
```

Comment:
You may not create view for the queries just do below thing directly in verify file : -
```
select 
        date_bucket(null, 2, cast('2020-01-01' as date)) as db
```